app/views/test/saml_test/decode_response.html.erb

```diff
@@ -26,6 +26,7 @@
         <%= link_to 'Open in New Window',
                     "data:text/xml;charset=utf-8;base64,#{xml_doc}",
                     target: '_blank', rel: 'noopener noreferrer' %>
+        <input type="hidden" id="SAMLResponse" value="<%= xml_doc %>">
```

Comment:
I know this is only used in tests, but I feel like `xml_doc` sounds like something we should escape
```suggestion
        <%= hidden_field_tag xml_doc, id: 'SAMLResponse' %>
```